.github/auto_assign.yml

```diff
@@ -0,0 +1,23 @@
+# Set to true to add reviewers to pull requests
+addReviewers: false
+
+# Set to true to add assignees to pull requests
+addAssignees: true
+
+# A list of assignees to be added to pull requests (GitHub user name)
+assignees:
```

Comment:
I'm guessing there's no way of making it pick someone at random from a Github team? Just thinking it might end up being a bit annoying to have _one more_ thing to do when someone joins/leaves the team...

Reply:
Indeed not.

Reply:
This would be an alternative, but we would lose the skipKeywords feature: would apply to all PRs. We would need to ask someone with more admin rights than me to do this. https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/managing-code-review-assignment-for-your-team

Reply:
Hmm, yeah. Another concern I'd have with random assignment (either through this or Github native functionality) is that the person it's assigned to might be on leave etc.

A more low-tech option (maybe aided by a Slack automation) could just be having a weekly rota of "first point of call" within the team, with a defined list of responsibilities including this, but also other things that are mildly annoying but should be shared within the team?

Reply:
Sounds good! We used to have a 'BAU' person for the sprint but the role wasn't very clearly defined so it died.

Reply:
Will keep PR open for another day or so in case anyone else wants to chime in.